src/restaurant_inspections_api/handlers/businesses.clj

```diff
@@ -0,0 +1,55 @@
+(ns restaurant-inspections-api.handlers.businesses
+  (:require [restaurant-inspections-api.validations :as validate]
+            [restaurant-inspections-api.util :as util]
+            [restaurant-inspections-api.services :as srv]))
+
+(defn validate-businesses-params
+  "Recieves all businesses query parameters (nil when not specified) and
+  returns a map of valid and invalid params"
+  [zip-codes county-number per-page page]
+
+  (let [validated-map {:zipCodes (validate/zip-codes zip-codes)
+                       :county (validate/county-number county-number)
+                       :perPage (validate/per-page per-page)
+                       :page (validate/page page)}]
+    {:invalid (into {} (filter #(false? (second %)) validated-map))
+     :valid (into {} (filter #(not (false? (second %)))) validated-map) }))
+
+(defn format-query-params-error
```

Comment:
wow this is the exact same function than the on in handlers/inspections. I think we could reuse this one (move to service.clj? or somewhere) and use it in both files.